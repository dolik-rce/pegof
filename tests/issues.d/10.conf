input issues.d/10.peg
optimize all
header never
