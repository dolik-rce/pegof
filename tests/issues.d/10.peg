name
    <- < [A-Z][A-Za-z0-9]* > {
        if (PEEK_KIND (auxil) != KIND_GHOST_INDEX)
            SET_SCOPE(auxil, makeVarlinkTag(auxil, $1, $1s));
    }
