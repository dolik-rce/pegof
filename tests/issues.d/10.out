name <-
    <[A-Z] [0-9A-Za-z]*> {
        if (PEEK_KIND (auxil) != KIND_GHOST_INDEX)
            SET_SCOPE(auxil, makeVarlinkTag(auxil, $1, $1s));
    }
